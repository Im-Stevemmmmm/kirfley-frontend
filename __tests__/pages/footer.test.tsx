import ReactDOM from 'react-dom';
import Footer from '../../components/footer';

it('renders correctly', () => {
  const container = document.createElement('div');
  ReactDOM.render(<Footer />, container);

  expect(container).toMatchSnapshot();
});
