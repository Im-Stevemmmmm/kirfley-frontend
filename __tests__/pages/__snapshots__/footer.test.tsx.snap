// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    id="container"
  >
    <footer
      id="footer"
    >
      <ul
        class="column"
      >
        <li>
          About
        </li>
        <li
          title="About the company behind Fern"
        >
          Company
        </li>
      </ul>
      <ul
        class="column"
      >
        <li>
          Support
        </li>
        <li
          title="Get help with any problems"
        >
          Contact Us
        </li>
      </ul>
      <ul
        class="column"
      >
        <li>
          Legal
        </li>
        <li
          title="Fern's privacy policy"
        >
          Privacy Policy
        </li>
        <li
          title="How we use cookies"
        >
          Cookies
        </li>
        <li
          id="copyright"
        >
          Â© Fern 
          2020
        </li>
      </ul>
    </footer>
  </div>
</div>
`;
